### PR TITLE
Manual install rabbitmq

### PR DESCRIPTION
**Pull request by EMaksy**
This PR will enrich the installation part of RabbitMQ by adding steps for a configuration according to the  official RabbitMQ suggestions.  Also, update the Docker commands in order to use the correct vhost.

### CODE REVIEW

File: guides/manual-installation.md [hunk 1]
```diff
@@ -142,59 +152,115 @@ Restart postgres to apply the changes:
...
 
-### Install rabbitmq
+### Install RabbitMQ
 
 ```bash
 zypper install rabbitmq-server
 ```
 
-Enable and start the rabbitmq service:
+As the agent needs to reach RabbitMQ, allow connections from external hosts.
```
Comment (rtorrero) [inline guides/manual-installation.md:161]: >As the agent needs to reach RabbitMQ, allow connections from external hosts.
As the agent needs to reach RabbitMQ, connections from external hosts need to be allowed?
wdyt?

File: guides/manual-installation.md [hunk 2]
```diff
@@ -65,7 +65,7 @@ systemctl enable --now prometheus
 Allow prometheus to be accessible from docker. Add an exception on firewalld:
 
 ```bash
-firewall-cmd --zone=public --add-port=9090/tcp --permanent
+firewall-cmd --zone=public --add-port=9090/tcp --permanent;
```
Comment (rtorrero) [inline guides/manual-installation.md:68]: The `;` doesn't hurt but it's not needed: if a user copy-pastes two lines, the newline char will also get copied and cause that two separate commands are executed.

File: guides/manual-installation.md [hunk 3]
```diff
@@ -243,7 +308,7 @@ docker run -d --name wanda \
     -e CORS_ORIGIN=localhost \
     -e SECRET_KEY_BASE=$WANDA_SECRET_KEY_BASE \
     -e ACCESS_TOKEN_ENC_SECRET=$ACCESS_TOKEN_ENC_SECRET \
-    -e AMQP_URL=amqp://guest:guest@host.docker.internal \
+    -e AMQP_URL=amqp://trento_user:trento_user_password@host.docker.internal/vhost\
```
Comment (rtorrero) [inline guides/manual-installation.md:311]: As we did it in all other lines, I'd put a space just before the final `\`

**Review by rtorrero — APPROVED** [review]
Some very small things (some of which just suggestions), but great work overall :+1: !

File: guides/manual-installation.md [hunk 4]
```diff
@@ -538,3 +538,8 @@ update-ca-certificates
 ```
 
 Configure trento using the `/etc/trento/agent.yaml` file and make sure to use `https` for the `server-url` parameter. Refer to https://documentation.suse.com/sles-sap/trento/html/SLES-SAP-trento/index.html#sec-trento-installing-trentoagent for more details.
+
+Additionally providing the correct RabbitMMQ user data and the correct vhost for facts-service-url is essential.
```
Comment (rtorrero) [inline guides/manual-installation.md:542]: RabbitMMQ -> RabbitMQ

**Review by CDimonaco — APPROVED** [review]

**Pull request merged**